crates/abq_cli/src/args.rs

```diff
@@ -326,6 +326,13 @@ pub enum Command {
         #[clap(long, default_value = "auto")]
         color: ColorPreference,
 
+        /// The maximum number of seconds to wait for a test command to start up.
+        ///
+        /// ABQ waits for the passed test command to start up before starting a test run.
```

Comment:
nitpick, but `passed test command` is a bit of a [garden-path phrase](https://en.wikipedia.org/wiki/Garden-path_sentence).

maybe
```suggestion
        /// ABQ waits for the test executable to start up before starting a test run.
```